Makefile

```diff
@@ -7,6 +7,10 @@ PROJECT_NAME := $(shell cat ./project_name.var)
 NEEDS_DEMO := $(shell cat ./needs_demo.var)
 BUILD_ID := $(shell git describe --always)
 
+# NOTE: The `DOCKER_IMAGE_BRANCH` tag is the git tag for the commit if it exists, else the branch on which the commit exists.
+# NOTE: Special characters in `DOCKER_IMAGE_BRANCH` are replaced with dashes.
+DOCKER_IMAGE_BRANCH := $(DOCKERHUB_REPO):$(shell git describe --exact-match --tags 2> /dev/null || git symbolic-ref --short HEAD | sed 's/[^[:alnum:]\.\_\-]/-/g')
```

Comment:
**Cool!** How about this be moved to the same block as the other `DOCKER_…` vars (lines 1–5)?

Reply:
Done